### DISCUSSION TITLE
Join documents

### DISCUSSION THREAD

**Discussion by Ostsol** [vkbo/novelWriter#1281] [Ideas]
There exists a feature whereby one may separate a document by its headers into individual documents. Could a feature be added to do the reverse? Simply: allow multiple selection in the Project tab and have a context menu option (and/or button and/or hotkey) to join the documents.

**Comment by vkbo** [comment]
It already exists. Any folder or document with child documents can be merged to one.

The project tree doesn't currently allow multi-select, so in order to merge documents you first have to move them into a folder. You can create a folder just for that purpose.

**Comment by Ostsol** [comment]
Thanks! It seems I used the wrong keyword when trying to search the manual.
